packages/style/__tests__/style.test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const style = require('..');
+
+describe('style', () => {
+    it('needs tests');
```

Comment:
Hmmm... not sure what tests to add here... checking for the existence of styles (in case of accidental deletion) seems a little arbitrary.  Really it seems like the components themselves should test for styles. Any thoughts on what's applicable here?